core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1562,8 +1562,14 @@ public void speedPresetChanged(SpeedPresetChangedEvent event) {
             if (((FeedMedia) getPlayable()).getItem().getFeed().getId() == event.getFeedId()) {
                 if (event.getSpeed() == SPEED_USE_GLOBAL) {
                     setSpeed(UserPreferences.getPlaybackSpeed(getPlayable().getMediaType()));
+                    skipSilence(UserPreferences.isSkipSilence());
```

Comment:
I know that the method was there before, but that's a weird method name. Could you please rename it to `setSkipSilence`?

Reply:
Done